tests/core/test_events.py

```diff
@@ -183,22 +184,49 @@ def test_json_parse_reminder():
     # fmt: off
     # DOCS MARKER ReminderScheduled
     evt = {
-        "event": "reminder",
-        "action": "my_action",
-        "date_time": "2018-09-03T11:41:10.128172",
-        "name": "my_reminder",
-        "kill_on_user_msg": True,
+      "event": "reminder",
+      "intent": "my_intent",
+      "entities": {"entity1": "value1", "entity2": "value2"},
+      "date_time": "2018-09-03T11:41:10.128172",
+      "name": "my_reminder",
+      "kill_on_user_msg": True,
     }
     # DOCS END
     # fmt: on
     assert Event.from_parameters(evt) == ReminderScheduled(
-        "my_action",
+        "my_intent",
         parser.parse("2018-09-03T11:41:10.128172"),
         name="my_reminder",
         kill_on_user_message=True,
     )
 
 
+def test_json_parse_reminder_cancelled():
+    # fmt: off
```

Comment:
do you actually need the `fmt: off`?

Reply:
I copied that from the other tests. I thought it has something to do with the fact that it is displayed in the Rasa Docs. All the other tests where `evt` does not fit into one line have this flag.

Reply:
As far as I know it prevents `black` from formatting it. You'd have to try out how the black formatting would look and if we actually need the `fmt: off`

Reply:
If I turn it off, the code gets indented more by BLACK. That may be problematic for small screens, and also inconsistent with other code in the docs.